packages/frontend/editor-ui/src/components/Evaluations.ee/shared/TestTableBase.vue

```diff
@@ -111,9 +115,17 @@ defineSlots<{
 		:default-sort="defaultSort"
 		:data="localData"
 		:border="true"
-		:cell-class-name="$style.customCell"
+		:cell-class-name="
+			({ row }) => {
+				return `${expandedRows?.has(row.id) ? $style.expandedCell : $style.baseCell}`;
+			}
+		"
 		:row-class-name="
-			({ row }) => (row?.status === 'error' ? $style.customDisabledRow : $style.customRow)
+			({ row }) => {
+				const baseClass = row?.status === 'error' ? $style.customDisabledRow : $style.customRow;
+				const expandedClass = expandedRows?.has(row.id) ? $style.expandedRow : '';
+				return `${baseClass} ${expandedClass}`;
+			}
```

Comment:
Extracting these functions and referencing them in parameter values might improve readability